README.md

```diff
@@ -67,6 +67,28 @@ This is done with the zenfs utility, using the mkfs command:
 ./plugin/zenfs/util/zenfs mkfs --zbd=<zoned block device> --aux_path=<path to store LOG and LOCK files>
 ```
 
+## Test
```

Comment:
Could you also update the `clang-format-11` calls in the `Contribution Guide` section to include the test files? :)